spread-tests/regression/lp-1606277/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Regression check for https://bugs.launchpad.net/snap-confine/+bug/1606277 
+# This is blacklisted on debian because debian doesn't use apparmor yet
+systems: [-debian-8]
+details: |
+    A missing bind mount for /var/log prevents access to system log files
+    even if the log-observe interface is being used.
+prepare: |
+    echo "Having installed the snapd-hacker-toolbelt snap"
+    snap install snapd-hacker-toolbelt
+    echo "And having connected the log-observe interface"
+    snap connect snapd-hacker-toolbelt:log-observe ubuntu-core:log-observe
+execute: |
+    echo "We can now see a non-empty /var/log directory"
+    cd /
+    [ "$(/snap/bin/snapd-hacker-toolbelt.busybox ls /var/log | wc -l)" != 0 ]
```

Comment:
Is non-empty the correct check? When I was triaging this issue I saw that there were things in /var/log due to the core snap having them there. Perhaps a better test would be for /var/log/kern.log or /var/log/syslog? Those might be Debian/Ubuntu specific though. You could look in /snap/ubuntu-core/current/var/log and compare the number of files there to the number in /var/log in the snap....